src/pages/index.js

```diff
@@ -62,13 +134,53 @@ const cardSection = new Section(
   ".cards__list",
 );
 
-cardSection.renderItems();
-
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addImageForm);
+const profileImageFormValidator = new FormValidator(config, profileImageForm);
+const profileEditFormValidator = new FormValidator(config, profileEditForm);
```

Comment:
`const profileEditFormValidator = new FormValidator(config, profileEditForm);
`

It's not needed because you have created a validator for the profile form in line 137